cfpb/consumerfinance.gov
# P4C: Add programs for undergrads

Exposes program-specific data for undergraduates as well as graduate students.


## Additions

- Display program select (when there are programs available) for undergrad program types
- Show program-specific text and salary data for undergrads



## How to test this PR

1. Check out branch and run gulp
2. Navigate to [financial path tool](https://dev2.demo.cfpb.gov/paying-for-college/your-financial-path-to-graduation/), select a college and an undergrad program type, and verify that program select is displayed
3. On the screens in the `Affording your loans` section, verify that the salary data displayed is program-specific


## Screenshots

### Program select for undergrads
<img width="733" alt="Screen Shot 2020-12-17 at 7 40 55 AM" src="https://user-images.githubusercontent.com/778171/102509975-028de080-403c-11eb-8955-8a00dbf855bf.png">

### Salary data when program is selected

<img width="637" alt="Screen Shot 2020-12-17 at 7 42 15 AM" src="https://user-images.githubusercontent.com/778171/102510109-1f2a1880-403c-11eb-9c90-6d5775d29165.png">

### Salary data when program is not selected

<img width="624" alt="Screen Shot 2020-12-17 at 7 41 44 AM" src="https://user-images.githubusercontent.com/778171/102510136-24876300-403c-11eb-9663-30d402fb8583.png">




## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
